agent/custom/bbc_action.py

```
@@ -635,8 +643,61 @@ def _wait_for_battle_end(self, state: dict):
+                        # 先覆盖"执行战斗结束"的 next 为"阶段判断循环"（确保覆盖必然生效），
+                        # 再停止BBC脚本：stop_battle 会触发"脚本停止"弹窗，避免其干扰后续流程
+                        # （助战错误的点击会使游戏重新进入登录状态，"结束战斗"流程无法走通）
+                        mfaalog.info("[ExecuteBbcTask] 覆盖'执行战斗结束'为阶段判断循环，处理游戏状态...")
+                        context.override_next("执行战斗结束", ["阶段判断循环"])
+                        # 停止BBC脚本，避免继续干扰模拟器操作
+                        stop_result = manager.send_command('stop_battle', {}, timeout=5)
+                        if stop_result.get('success'):
+                            mfaalog.info("[ExecuteBbcTask] BBC脚本已停止")
+                        else:
+                            mfaalog.warning(f"[ExecuteBbcTask] 停止BBC脚本失败: {stop_result.get('error', '')}")
+                        # 等待BBC脚本完全停止，避免干扰后续任务
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)